callbacks.go

```diff
@@ -183,3 +185,17 @@ func (exec *sidecarExecutor) Shutdown(driver executor.ExecutorDriver) {
 func (exec *sidecarExecutor) Error(driver executor.ExecutorDriver, err string) {
 	log.Info("Got error message:", err)
 }
+
+// Check if it should check Sidecar status, assuming enabled by default
+func shouldCheckSidecar(containerConfig *docker.CreateContainerOptions) bool {
```

Comment:
I know I probably said to pass the container config here, but it would make more sense to pass the `mesos.TaskInfo`. We can then remove the Docker import here, too, I think,